ext/native-decls/GetAllPeds.md

```diff
@@ -16,3 +16,36 @@ The data returned adheres to the following layout:
 
 ## Return value
 An object containing a list of peds handles.
+
+## Examples
+
+```lua
+--[[ 
+Example =>
+  Using GetAllPeds, This example prints information of all peds, Ped id, Owner, Model and Armour.
+]]
+
+-- This useful function GetPedOwner returns the player that owned the given ped.
+function GetPedOwner(ped)
```

Comment:
This function is redundant, just use [NetworkGetEntityOwner](https://docs.fivem.net/natives/?_0x526FEE31)